client/src/static_data/campaign_creator.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Nitpicky, but the domain "static_data" isn't super descriptive to me.  I'd prefer namespaces like "src/assets/campaign_data" personally, but I'm curious what Tom has to say about it